src/App.js

```diff
@@ -44,6 +47,6 @@ export default function App() {
           </Route>
```

Comment:
a nice to have:
`        <Route path="*">
          <Redirect to="/" />
        </Route>`
and you have to import Redirect from react-router-dom -
`import {
  BrowserRouter as Router,
  Switch,
  Route,
  Redirect,
} from "react-router-dom";`
This way all invalid request like /blub will get redirect to your home / path ;)